hawkular-alerts-engine/pom.xml

```diff
@@ -169,15 +187,84 @@
 
   <profiles>
 
+    <!-- Profiles to compile with specific clustering libraries -->
     <profile>
-      <id>default</id>
+      <id>clustering-wildfly</id>
       <activation>
-        <activeByDefault>true</activeByDefault>
+        <property>
+          <name>!env</name>
+        </property>
       </activation>
+      <dependencies>
+        <dependency>
+          <groupId>org.infinispan</groupId>
+          <artifactId>infinispan-core</artifactId>
+          <scope>provided</scope>
+          <version>${version.org.infinispan.wildfly}</version>
+        </dependency>
+      </dependencies>
       <build>
         <plugins>
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>default-compile</id>
+                <phase>compile</phase>
+                <goals>
+                  <goal>compile</goal>
+                </goals>
+                <configuration>
+                  <excludes>
+                    <exclude>org/hawkular/alerts/engine/impl/PartitionManagerEapImpl.java</exclude>
```

Comment:
No, if I can avoid it.
In my first attempt I saw that ISPN/JGroups versions are very different, and far in versions, so I introduce the conditional compilation for this case.
But after study and make some tests, I could use the ISPN 5 API that is present in both EAP and Wildfly versions.
So no conditional compilations needed so far.
I forgot to change this one.
Good catch.